lib/sim.c

```diff
@@ -562,6 +569,7 @@ sim_add_pixel_to_occupancy(sim_t *self, unsigned int h, unsigned int pixel)
             goto out;
         }
         if (avl_insert_node(tree, h_node) == NULL) {
+            printf("Insert node error");
```

Comment:
This is odd. I can see the line in my Github repository, but not in the sim.c file in my local branch. I'll try and investigate how it got here.